Classes/DataProcessing/LanguageMenuProcessor.php

```diff
@@ -20,23 +21,20 @@
  * decoded again and assigned to FLUIDTEMPLATE as variable.
  *
  * Options:
- * alternativeSortingField -
```

Comment:
why do you remove the sorting support?

Reply:
sorting has been removed since master and 9.3 implementation does currently not support this - to avoid confusions when people are upgrading a feature could be missing. the goal is to polyfill the 9.3 behavior,

Reply:
that's fine, we could readd it if sorting of core is done if desired.